### PR TITLE
[RHCLOUD-23955] update Patch email templates according UXD guidelines

### PR DESCRIPTION
**Pull request by g-duval**
image

image

### CODE REVIEW

**Review by josejulio — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java [hunk 1]
```diff
@@ -82,6 +82,9 @@ public class FeatureFlipper {
     @ConfigProperty(name = "notifications.emails-only-mode.enabled", defaultValue = "false")
     boolean emailsOnlyMode;
 
+    @ConfigProperty(name = "notifications.use-parch-email-templates-v2.enabled", defaultValue = "false")
```
[the remark this answers is not in the document]
Comment (g-duval) [reply common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java:85]: Oups... thanks

File: common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java [hunk 2]
```diff
@@ -214,6 +218,14 @@ public void setEmailsOnlyMode(boolean emailsOnlyMode) {
         this.emailsOnlyMode = emailsOnlyMode;
     }
 
+    public boolean isPatchEmailTemplatesV2Enabled() {
+        return patchEmailTemplatesV2Enabled;
+    }
+
+    public void setPatchEmailTemplatesV2Enabled(boolean patchEmailTemplatesV2Enabled) {
+        this.patchEmailTemplatesV2Enabled = patchEmailTemplatesV2Enabled;
```
[the remark this answers is not in the document]
Comment (g-duval) [reply common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java:226]: Ok

File: engine/src/main/resources/templates/Common/insightsEmailBody.html [hunk 3]
```diff
@@ -276,8 +276,7 @@
             padding: 10px;
             text-align: left;
             font-size: 14px;
-            font-weight: 500;
-            text-transform: uppercase;
+            font-weight: bold;
```
[the remark this answers is not in the document]
Comment (g-duval) [reply engine/src/main/resources/templates/Common/insightsEmailBody.html:279]: It have to impact all templates, I didn't noticed that when I added this common file bases on Policies, but according UXD mockups, table headers have to be Bold, but not full uppercase.

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**